engine/launcher/args/args.go

```diff
@@ -136,3 +153,12 @@ func (args EngineServerArgs) validate() error {
 	}
 	return nil
 }
+
+func contains(slice []string, value string) bool {
```

Comment:
you can remove this helper function if you use a set instead

Reply:
thanks! this makes sense :)